jans-cli-tui/cli_tui/plugins/999_jans/main.py

```
@@ -106,3 +128,139 @@ async def coroutine():
+        ])
+
+        create_client_button_label = _("Create Client")
+        buttons = [
+            Button(create_client_button_label, handler=self.create_ssa_cli, width=len(create_client_button_label)+4),
+            Button(_("Cancel"))
+        ]
+        dialog = JansGDialog(self.app, title=_("SSA for Creating Client"), body= body, buttons=buttons)
+        self.app.show_jans_dialog(dialog)
+
+    def clear_config(self):
+        def do_clear_config(dialog):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `do_clear_config`

Add return type annotation: `None`

(ANN202)